src/ui/public/kuery/lib/functions/and.js

```diff
@@ -0,0 +1,46 @@
+import * as ast from '../ast';
+import { nodeTypes } from '../node_types';
+
+export function buildNodeParams(children, serializeStyle = 'implicit') {
+  return {
+    arguments: children,
+    serializeStyle
+  };
+}
+
+export function toElasticsearchQuery(node, indexPattern) {
+  const children = node.arguments || [];
+
+  return {
+    bool: {
+      filter: children.map((child) => {
+        if (child.type === 'literal') {
+          child = nodeTypes.function.buildNode('is', '*', child.value);
```

Comment:
Hm, or we could introduce an "implicit" style for `is`, which would preserve that formatting while still having an unambiguous AST. I don't have enough experience with building languages to judge the pros and cons of either solution. My intuition is that we should either create a fully "interpreted" AST as early as possible or as late as possible, but not in between.